scripts/generate_contributors_list.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+ContributersPath=../CONTRIBUTORS.txt
```

Comment:
Nit: there is a typo in this variable name.

Reply:
Good catch, I have updated it now...